src/effects/VST/VSTEffect.cpp

```diff
@@ -1341,7 +1355,8 @@ std::unique_ptr<EffectUIValidator> VSTEffect::PopulateUI(ShuttleGui &S,
 
    auto pParent = S.GetParent();
    pParent->PushEventHandler(this);
-   return std::make_unique<DefaultEffectUIValidator>(*this, access, pParent);
+
+   return std::make_unique<VSTEffectValidator>(*this, access, pParent);
```

Comment:
The minimal validator class doesn't yet appear to be necessary for the VST2 transformation, unlike the case for AudioUnits and VST3.  But no harm in keeping the commit that adds it.

Reply:
Well actually at this point I need it, because I pass the instance it belongs to in its constructor, to retrieve it later in ::ShowClientInterface